libs/types/src/tokens.rs

```diff
@@ -41,23 +51,24 @@ use crate::{xcm::XcmMetadata, EVMChainId};
 pub enum CurrencyId {
 	// The Native token, representing AIR in Altair and CFG in Centrifuge.
 	#[default]
+	#[codec(index = 0)]
 	Native,
 
-	// A Tranche token
+	/// A Tranche token
+	#[codec(index = 1)]
 	Tranche(PoolId, TrancheId),
 
-	/// Karura KSM
-	KSM,
-
-	/// Acala Dollar
-	/// In Altair, it represents AUSD in Kusama;
-	/// In Centrifuge, it represents AUSD in Polkadot;
+	#[codec(index = 2)]
+	/// DEPRECATED - Will be removed in the following up Runtime Upgrade once
+	/// the orml_tokens' balances are migrated to the new CurrencyId for AUSD.
```

Comment:
`KSM` can be removed as we don't have balances for that, right?

Reply:
Yap, this is just in the temporarily `mod before` to have the exact same previous definition of `CurrencyId` and ensure the other variants encode the way we expect them to in the new representation.

Reply:
The `AUSD` variant should be assigned index `3` as `2` currently maps to `KSM`. We must leave a note that both indices are deprecated.